python/ray/rllib/utils/actors.py

```diff
@@ -53,6 +53,18 @@ def completed_prefetch(self):
                 remaining.append((worker, obj_id))
         self._fetching = remaining
 
+    def reset_evaluators(self, evaluators):
```

Comment:
This is needed since otherwise a blacklisted evaluator might still have old tasks in the TaskPool object.